From f87f6b5d80d5933d25bc179a5eab5c1e2c1a28ba Mon Sep 17 00:00:00 2001
From: Till Kampetter <till.kampetter@gmail.com>
Date: Tue, 9 Aug 2016 18:11:35 +0200
Subject: Do not use host names for broadcasting print queues and

 managing print queues broadcasted from other servers by default. Many
 networks do not have valid host names for all machines.
Bug-Ubuntu: https://bugs.launchpad.net/bugs/449586

Patch-Name: do-not-broadcast-with-hostnames.patch
---
 scheduler/conf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scheduler/conf.c b/scheduler/conf.c
index 25eeb218c..6bcbb262b 100644
--- a/scheduler/conf.c
+++ b/scheduler/conf.c
@@ -899,7 +899,7 @@ cupsdReadConfiguration(void)
     cupsdAddAlias(ServerAlias, temp);
     cupsdLogMessage(CUPSD_LOG_DEBUG, "Added auto ServerAlias %s", temp);
 
-    if (HostNameLookups || RemotePort)
+    if (HostNameLookups)
     {
       struct hostent	*host;		/* Host entry to get FQDN */
 
